composer.json

```diff
@@ -11,17 +11,17 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/vendor-plugin": "^1.0",
-        "dnadesign/silverstripe-elemental": "^4.7",
-        "silverstripe/elemental-fileblock": "^2@dev",
-        "silverstripe/cms": "^4.3",
-        "silverstripe/admin": "^1.7@dev",
-        "silverstripe/framework": "^4.10"
+        "php": "^8.1",
+        "silverstripe/admin": "^2",
+        "silverstripe/cms": "^5",
+        "silverstripe/framework": "^5",
+        "silverstripe/elemental-fileblock": "^3",
+        "silverstripe/vendor-plugin": "^2",
+        "dnadesign/silverstripe-elemental": "5.0.0-alpha2"
```

Comment:
@sabina-talipova why https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/33 requires `^5` and here we require `5.0.0-alpha2`?

Reply:
@michalkleiner, It's a temporary solution to pass GA with installation of lowest available version. We'll fix it after final release.